includes/class-block-patterns.php

```diff
@@ -0,0 +1,202 @@
+<?php
+/**
+ * The Block Patterns post type.
+ *
+ * @package CoBlocks
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * The Block Patterns post type.
+ *
+ * @since NEXT
+ */
+class CoBlocks_Block_Patterns {
+	/**
```

Comment:
@jrtashjian this would be a good place to declare `const POST_TYPE = 'coblocks_pattern';` then just use `self::POST_TYPE` throughout the class

Reply:
@jrtashjian same could also be done for `CATEGORY_TAXONOMY` and `TYPE_TAXONOMY`

Reply:
Updated in 091efc4f12b2d03c7788fda5fc9e035255523d8c